.gitlab/benchmarks/macrobenchmarks.yml

```
@@ -362,3 +368,26 @@ attack-appsec-enabled-iast-enabled-ep-enabled:
     DD_APPSEC_RASP_ENABLED: "true"
     IAST_WORST: "true"
     DD_SERVICE: "ddtrace-mb-attack-appsec-enabled-iast-enabled-ep-enabled"
+
+check-slo-breaches:
+  stage: gate
+  when: always
+  tags: ["arch:amd64"]
+  image: registry.ddbuild.io/images/benchmarking-platform-tools-ubuntu:latest
+  artifacts:
+    name: "artifacts"
+    when: always
+    paths:
+      - platform/artifacts/
+    expire_in: 3 months
+  script:
+    - cd platform && (git init && git remote add origin https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform && git pull origin erikayasuda/prereleasegates)
+    - bp-runner bp-runner.fail-on-breach.yml
+
+notify-slo-breaches:
+  extends: .notify-slo-breaches
+  stage: notify
+  needs: ["check-slo-breaches"]
+  when: always
+  variables:
+    CHANNEL: "test-ddtracepy-alerts"
```

TODO: Change this back to release channel after testing
```suggestion
    CHANNEL: "apm-python-release"
```